main.py

```diff
@@ -71,3 +72,17 @@ async def stop_mtp(self):
         if not is_running():
             return
         subprocess.run("./stop.sh", cwd=PLUGIN_BIN_DIR, shell=True)
+
+    # Toggle eth
+    async def toggle_eth(self) -> bool:
+        if not is_running():
+            subprocess.run("./usbeth.sh start -r", cwd=PLUGIN_BIN_DIR, shell=True)
+        else:
+            subprocess.run("./usbeth.sh stop", cwd=PLUGIN_BIN_DIR, shell=True)
+        return is_running()
+        
+    # Stop eth
+    async def stop_eth(self):
+        if not is_running():
+            return
+        subprocess.run("sudo ./usbeth.sh stop", cwd=PLUGIN_BIN_DIR, shell=True)
```

Comment:
Don't need sudo here, the plugin is running as root.

Reply:
just copy pasted from what i was typing in knosole